app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -532,11 +525,11 @@ bool OpenXRInputSource::GetHandTrackingInfo(const XrFrameState& frameState, XrSp
     XrHandJointLocationsEXT jointLocations { XR_TYPE_HAND_JOINT_LOCATIONS_EXT };
     jointLocations.jointCount = XR_HAND_JOINT_COUNT_EXT;
     jointLocations.jointLocations = mHandJoints.data();
-    jointLocations.next = &mAimState;
+    jointLocations.next = isFBHandTrackingAimExtensionEnabled ? &mAimState : nullptr;
 
     CHECK_XRCMD(OpenXRExtensions::sXrLocateHandJointsEXT(mHandTracker, &locateInfo, &jointLocations));
     mHasHandJoints = jointLocations.isActive;
-    mHasAimState = (mAimState.status & XR_HAND_TRACKING_AIM_VALID_BIT_FB) != 0;
+    mHasAimState = isFBHandTrackingAimExtensionEnabled ? (mAimState.status & XR_HAND_TRACKING_AIM_VALID_BIT_FB) : false;
```

Comment:
I don't understand this change, is it failing for Lynx?
Normally when a runtime doesn't support a structure in the chain it should ignore it.

Reply:
Right, it isn't strictly needed. I just thought it was a good step towards removing the dependency with the extension but it's true that the runtime should ignore it.

I'll just remove the early return.